packages/app/src/components/views/publication/PreviewPostView.tsx

```diff
@@ -0,0 +1,158 @@
+    setTags(currentTags)
+  }
+
+  return (
+    <PublicationPage publication={publication} showCreatePost={false}>
+      <ViewContainer maxWidth="sm">
+        <form onSubmit={handleSubmit((data) => onSubmitHandler(data as { description: string }))}>
+          <Grid container gap={4} flexDirection="column" mt={12.5}>
+            <Grid item>
+              <Grid container justifyContent={"space-between"} alignItems="center">
+                <Typography
+                  color={palette.grays[1000]}
+                  variant="h5"
+                  fontFamily={typography.fontFamilies.sans}
+                  sx={{ margin: 0 }}
+                >
+                  Post Preview
+                </Typography>
+                <CloseIcon style={{ cursor: "pointer" }} onClick={() => navigate(-1)} />
+              </Grid>
+            </Grid>
+            <Grid item>
+              <UploadFile onFileSelected={setArticleImg} />
+            </Grid>
+            <Grid item>
+              <Typography
+                color={palette.grays[800]}
+                variant="h5"
+                fontFamily={typography.fontFamilies.sans}
+                sx={{ margin: 0 }}
+              >
+                {publication?.title}
+              </Typography>
+            </Grid>
+
+            <Grid item>
+              <Controller
+                control={control}
+                name="description"
+                render={({ field }) => (
+                  <TextField
+                    {...field}
+                    placeholder="Write a brief description..."
+                    sx={{ width: "100%" }}
+                    multiline
+                    rows={4}
+                  />
+                )}
+              />
+            </Grid>
+            <Grid item>
+              <Typography
+                color={palette.grays[1000]}
+                variant="h6"
+                fontFamily={typography.fontFamilies.sans}
+                sx={{ margin: 0 }}
+              >
+                Add up to 5 tags so your readers know what this post is about:
+              </Typography>
+            </Grid>
+            <Grid item>
+              <TextField
+                value={tag}
+                onChange={({ target }) => setTag(target.value)}
+                sx={{ width: "100%" }}
+                placeholder="Add a tag..."
+                onKeyPress={handleTagKeyEvent}
+              />
+            </Grid>
+            <Grid item>
+              <Grid container gap={1}>
+                {tags.map((name, index) => (
+                  <Chip label={name} size="small" key={index} onDelete={() => handleDeleteTag(index)} />
+                ))}
+              </Grid>
+            </Grid>
+
+            <Grid item xs={12}>
+              <Button variant="outlined" size="large" type="submit" disabled={loading}>
+                {loading && <CircularProgress size={20} sx={{ marginRight: 1 }} />}
+                Publish now
+              </Button>
```

Comment:
Same mouseover error as on the Publish button in `packages/app/src/components/views/publication/CreatePostView.tsx`.